### ISSUE TITLE
IN-READTABLE return value is inconsistent with IN-PACKAGE

### ISSUE THREAD

**Issue by phoe** [melisgl/named-readtables#13]
```common-lisp
CL-USER (in-package #:gateway.engine/test)
#<PACKAGE "GATEWAY.ENGINE/TEST">
ENGINE/TEST> (in-readtable protest/parachute)
(("GATEWAY.ENGINE/TEST" . #<NAMED-READTABLE PROTEST/PARACHUTE {101BA00863}>)
 ("PROTEST/FOR-PARACHUTE" . #<NAMED-READTABLE PROTEST/PARACHUTE {101BA00863}>)
 ("SB-VM" . #<READTABLE {101BA008B3}>)
 ("SB-FORMAT" . #<READTABLE {101BA008B3}>)
 ("SB-PROFILE" . #<READTABLE {101BA008B3}>)
 ("SB-ALIEN" . #<READTABLE {101BA008B3}>) ("SB-C" . #<READTABLE {101BA008B3}>)
 ("SB-X86-64-ASM" . #<READTABLE {101BA008B3}>)
 ("SB-IMPL" . #<READTABLE {101BA008B3}>)
 ("SB-REGALLOC" . #<READTABLE {101BA008B3}>)
 ("SB-SYS" . #<READTABLE {101BA008B3}>)
 ("SB-THREAD" . #<READTABLE {101BA008B3}>)
 ("SB-BIGNUM" . #<READTABLE {101BA008B3}>)
 ("SB-GRAY" . #<READTABLE {101BA008B3}>)
 ("SB-LOOP" . #<READTABLE {101BA008B3}>)
 ("SB-KERNEL" . #<READTABLE {101BA008B3}>)
 ("SB-POSIX" . #<READTABLE {101BA008B3}>)
 ("SB-EXT" . #<READTABLE {101BA008B3}>)
 ("SB-ALIEN-INTERNALS" . #<READTABLE {101BA008B3}>)
 ("SB-DISASSEM" . #<READTABLE {101BA008B3}>)
 ("SB-BSD-SOCKETS-INTERNAL" . #<READTABLE {101BA008B3}>)
 ("SB-MOP" . #<READTABLE {101BA008B3}>)
 ("SB-PRETTY" . #<READTABLE {101BA008B3}>)
 ("SB-WALKER" . #<READTABLE {101BA008B3}>)
 ("SB-INT" . #<READTABLE {101BA008B3}>) ("SB-UNIX" . #<READTABLE {101BA008B3}>)
 ("SB-ASSEM" . #<READTABLE {101BA008B3}>)
 ("SB-APROF" . #<READTABLE {101BA008B3}>)
 ("SB-EVAL" . #<READTABLE {101BA008B3}>)
 ("SB-CLTL2" . #<READTABLE {101BA008B3}>)
 ("SB-UNICODE" . #<READTABLE {101BA008B3}>)
 ("SB-DEBUG" . #<READTABLE {101BA008B3}>)
 ("SB-PCL" . #<READTABLE {101BA008B3}>)
 ("SB-BSD-SOCKETS" . #<READTABLE {101BA008B3}>)
 ("SB-SEQUENCE" . #<READTABLE {101BA008B3}>)
 ("SB-FASL" . #<READTABLE {101BA008B3}>)
 ("SB-INTROSPECT" . #<READTABLE {101BA008B3}>)
 ("SB-DI" . #<READTABLE {101BA008B3}>))
```

I do not think that IN-READTABLE needs to be *this* verbose with its return value. If it is analogous to IN-PACKAGE, why does it not return the `#<NAMED-READTABLE PROTEST/PARACHUTE {101BA00863}>`?

**Comment by melisgl** [comment]
Thanks. Fixed in 78f987ac46f09a087b0a1ffc24078e2984864eee.

**Comment by phoe** [comment]
Woo! Thanks.
